.circleci/config.yml

```diff
@@ -129,7 +129,9 @@ workflows:
               only:
                 - master
       - release:
-          context: github-fin-eng-context
+          context: 
+            - github-fin-eng-context
+            - npn-publish
```

Comment:
npm?

Reply:
it's not my context but i think they named it incorrectly...